content/docs/user-guide/interactivity/robotics/concepts-and-components-overview.md

```diff
@@ -0,0 +1,107 @@
+
+### Robot Control
+
+The Gem comes with `ROS2RobotControlComponent`, which you can use to move your robot through:
+
+- [Twist](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Twist.msg) messages.
+- [AckermannDrive](https://github.com/ros-drivers/ackermann_msgs/blob/master/msg/AckermannDrive.msg)
+  The component subscribes to these command messages on a configured topic. The topic is `cmd_vel` by default, in a namespace as dictated by __ROS2Frame__.
+
+To make use of received command messages, use either `AckermannControlComponent`, `RigidBodyTwistControlComponent`, or `SkidSteeringControlComponent`, depending on the steering type. You can also implement your own control component or use LUA scripting to handle these commands. Unless scripting is used, control components should translate ROS 2 commands to events on `VehicleInputControlBus`. These events will be handled by a [`VehicleModelComponent`](#vehicle-model) if it is present. You can use tools such as [rqt_robot_steering](https://index.ros.org/p/rqt_robot_steering/) to move your robot with Twist messages. `RobotControl` is suitable to use with [ROS 2 navigation stack](https://navigation.ros.org/). It is possible to implement your own control mechanisms with this component.
+
+### Vehicle Model
+
+`VehicleModelComponent` serves the purpose of converting inputs such as target velocity, steering or acceleration to physical forces on parts of a vehicle (robot). `VehicleModel` has a `VehicleConfiguration` which is used to define axles, parametrize and assign wheels. The model requires a `WheelControllerComponent` present in each wheel entity. It also uses an implementation of `DriveModel`, which converts vehicle inputs to forces acting on steering elements and wheels.
+
+### Vehicle Dynamics
+
+See the [Vehicle Dynamics](vehicle-dynamics.md) section.
+
+### Spawner
+
+`ROS2SpawnerComponent` handles spawning entities during simulation. Available spawnables have to be set up as the component's field before the simulation. User is able to define named spawn points inside the Editor. This can be done by adding `ROS2SpawnPointComponent` to a child entity of an entity with `ROS2SpawnerComponent`. During the simulation user can access names of available spawnables and request spawning using ros2 services. The names of services are `/get_available_spawnable_names` and `/spawn_entity` respectivly. GetWorldProperties.srv and SpawnEntity.srv types are used to handle these features. In order to request defined spawn points names user can use `/get_spawn_points_names` service with `GetWorldProperties.srv` type. Detailed information about specific spawn point (e.g. pose) can be accessed using `/get_spawn_point_info` service with `GetModelState.srv` type. All used services types are defined in gazebo_msgs package.
+
+- **Spawning**: The spawnable name must be passed in `request.name` and the position of entity in `request.initial_pose`.
```

Comment:
For instructions where the user is taking action, use an [active voice](https://writing.wisc.edu/handbook/style/ccs_activevoice/) ("you") instead of a passive voice.  It clarifies that user is the one taking action. This is especially important to distinguish from actions that happen automatically, or that the users should observe rather than perform.
```suggestion
- **Spawning**: To spawn, you must pass in the spawnable name into `request.name` and the position of entity into `request.initial_pose`.
```